configuration/controller/pacemaker.sh

```diff
@@ -167,9 +167,8 @@ if flag_is_set PRIMARY_INSTALL ; then
 
     # Primary core resources
     pcs -f $tmpfile resource create primary ocf:heartbeat:Dummy op monitor interval=179s
-    pcs -f $tmpfile resource create fs-ready ocf:heartbeat:Dummy op monitor interval=183s
```

Comment:
Why do you delete this? Please read the HA design documentation for the reason for this dummy resource.